### DISCUSSION TITLE
Parent Object

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7677] [General]
I have a set of configurations for products that will be different for each product. I would just prefer to send 1 object called config and that would contain whatever configs are in there for the product.

I need send this to a `vue.js` component. As in my vue compoent I can just loop through the key vale pairs, but right now I need to do a bunch of if and else statements as this will depend on what type of product config is selected.

right now the key-values come back as flat. How can I put these in an parent object?

```
size = small,
color = red,
brand = nike
```

```
configs:{
    size = small,
    color = red,
    brand = nike
}
```

**Comment by edalzell** [comment]
How are you sending things to Vue?
